spec/std/system/group_spec.cr

```diff
@@ -0,0 +1,76 @@
+require "spec"
+require "system/group"
+require "system/user"
+
+module Spec::Group
+  {% if flag?(:darwin) || flag?(:openbsd) || flag?(:freebsd) %}
+    NAME_STRING = "wheel"
+    NAME_ID = 0
+    PROP_MEMBERS = ["root"]
+  {% elsif flag?(:linux) %}
+    NAME_STRING = "root"
+    NAME_ID = 0
+    PROP_MEMBERS = [] of String
+  {% else %}
+    NAME_STRING = "root"
+    NAME_ID = 0
+    PROP_MEMBERS = [] of String
+  {% end %}
+
+  BAD_NAME_STRING = "non_existant_group"
+  BAD_NAME_ID     = 123456
+end
+
+describe System::Group do
+  it "groupname from gid" do
+    System::Group.name(Spec::Group::NAME_ID).should eq(Spec::Group::NAME_STRING)
+  end
+
+  it "gid from groupname" do
+    System::Group.gid(Spec::Group::NAME_STRING).should eq(Spec::Group::NAME_ID)
+  end
+
+  it "group existence" do
+    System::Group.exists?(Spec::Group::NAME_STRING).should be_true()
+    System::Group.exists?(Spec::Group::NAME_ID).should be_true()
+
+    System::Group.exists?(Spec::Group::BAD_NAME_STRING).should_not be_true()
+    System::Group.exists?(Spec::Group::BAD_NAME_ID).should_not be_true()
+  end
+
+  it "creates a group" do
+    System::Group.get(Spec::Group::NAME_STRING).should_not be_nil()
```

Comment:
Brackets with empty parameters are usually omitted. Ditto for the following lines.